test/e2e/auth/service_accounts.go

```diff
@@ -410,4 +415,143 @@ var _ = SIGDescribe("ServiceAccounts", func() {
 			}
 		}
 	})
+
+	//It("should support InClusterConfig with token rotation [Slow]", func() {
+	It("should support InClusterConfig with token rotation", func() {
```

Comment:
This test works now but its slow (~10 minutes). It's pretty gross because I want it to run on non-alpha clusters. I could drop a bunch of stuff if we are ok with this only running in the alpha test suite.

Reply:
Also it will still be slow even if we only run it in the alpha suite, just way smaller.

Reply:
I don't think we check feature gates in the e2e process (it's not even the same process as the apiserver)... typically we indicate non-default features with feature tags in the test description

Reply:
Ya, fixed.